command/src/state.rs

```diff
@@ -1069,8 +1069,8 @@ mod tests {
     println!("expected diff orders:\n{:#?}\n", e);
     println!("diff orders:\n{:#?}\n", diff);
 
-    let hash1 = state.hash_state();
-    let hash2 = state2.hash_state();
+    let _hash1 = state.hash_state();
```

Comment:
instead of removing the warning, let's think about why those variables were there and are unused. It's probably missing an `assert_eq!(hash1, hash2)`